x-pack/plugins/search_playground/public/chat_playground_overview.tsx

```diff
@@ -27,46 +25,18 @@ export const ChatPlaygroundOverview: React.FC = () => {
   );
 
   return (
-    <PlaygroundProvider>
-      <EuiPageTemplate
-        offset={0}
-        restrictWidth={false}
-        data-test-subj="svlPlaygroundPage"
-        grow={false}
-      >
-        <EuiPageTemplate.Header
-          css={{ '.euiPageHeaderContent > .euiFlexGroup': { flexWrap: 'wrap' } }}
-          pageTitle={
-            <EuiFlexGroup>
-              <EuiFlexItem grow={false}>
-                <EuiTitle
-                  css={{ whiteSpace: 'nowrap' }}
-                  data-test-subj="chat-playground-home-page-title"
-                >
-                  <h2>
-                    <FormattedMessage
-                      id="xpack.searchPlayground.pageTitle"
-                      defaultMessage="Playground"
-                    />
-                  </h2>
-                </EuiTitle>
-              </EuiFlexItem>
-              <EuiFlexItem grow={false}>
-                <EuiBetaBadge
-                  label={i18n.translate('xpack.searchPlayground.pageTitle.techPreview', {
-                    defaultMessage: 'TECH PREVIEW',
-                  })}
-                  color="hollow"
-                />
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          }
-          data-test-subj="chat-playground-home-page"
-          rightSideItems={[<PlaygroundHeaderDocs />, <PlaygroundToolbar />]}
-        />
-        <App />
-        {embeddableConsole}
-      </EuiPageTemplate>
-    </PlaygroundProvider>
+    <QueryClientProvider client={queryClient}>
```

Comment:
why not put the QueryClient as part of the PlaygroundProvider?

Reply:
I moved it here to use useSourceFields hook in the playground provider, to avoid bugs when the hook is unmounted and the index is changed. This hook used queryClient so I moved it to the parent component. But then I figured out that it affected a lot more and needed extra time to fix some bugs so I returned changes. I forgot to return this one as well. I can revert these changes

Reply:
I forgot that i've added a load connectors hook in playground provider, so query client provider needs to be defined in upper component

Reply:
do you think it still could be incapsulated in the PlaygroundProvider still? just the PlaygroundProvider is two components, for the hook and the queryClientProvider?

Reply:
if its not possible, i would just create a new provider component for QueryClient thats in our providers folder vs making the QueryClient instantiation in a util.

Reply:
we can add queryClientProvide in PlaygroundProvide, but in that case need to find another place to load connector and set models